src/exambank/exambank.py

```diff
@@ -0,0 +1,112 @@
+from io import BytesIO
+from pathlib import Path
+
+from pymupdf import Matrix
+from pymupdf import open as p_open
```

Comment:
I would prefer doing import pymupdf, then `pymupdf.open(...)`, but it's up to you~!

Reply:
Sure, 5f9895b.